src/app/collections/transparencyAttestations/transparencyAttestations.controller.spec.js

```diff
@@ -46,10 +46,17 @@
                 expect(vm._urlTransform(data)).toBe('Not available');
             });
 
-            it('should wrap the data in link text', function () {
+            it('should wrap the data in link text and not add http://', function () {
```

Comment:
This test depends on the use of utilService. We generally try to mock out any services that are used when testing controllers; see all of the mocks of networkService. If anything, I'd have changed the "data" array to have a "valid" url